packages/core/src/cli/perform-build.ts

```diff
@@ -15,14 +15,11 @@ interface BuildOptions extends TS.BuildOptions {
 export function performBuild(ts: TypeScript, projects: string[], buildOptions: BuildOptions): void {
   let transformManagerPool = new TransformManagerPool(ts.sys);
   let formatDiagnostic = buildDiagnosticFormatter(ts);
+  let buildProgram = ts.createEmitAndSemanticDiagnosticsBuilderProgram;
```

Comment:
In `perform-watch` below, you just inlined this; any reason not to do the same here?

Reply:
It made Prettier wrap the `patchProgramBuilder` call from 1 line to 5, and I... didn't like that 😅

Reply:
Hahahaha I figured it might be that. 👍🏼